### PR TITLE
Thuang dropdown item multi select columns

### PR DESCRIPTION
**Pull request by tihuan**
Screen Shot 2021-03-11 at 4 43 53 PM

demo

### CODE REVIEW

File: src/core/styles/common/mixins/body.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { css } from "@emotion/react";
+import { FontWeights } from "../constants/fontWeights";
+
+export const fontXs = css`
```
Comment (tihuan) [inline src/core/styles/common/mixins/body.ts:4]: @hthomas-czi this is an example of defining a mixin!

Comment (tihuan) [reply src/core/styles/common/mixins/body.ts:4]: And this is how to access the theme object in Emotion 😄 

```ts
const StyledAlert = styled(Alert)`
  ${shadows}
  ${palette}
  ${({ theme }) => {
    console.log("!!!! GOT THEME", theme);
    return `
      background-color: ${theme.palette.success.light}
    `;
  }}
`;
```

https://codesandbox.io/s/material-demo-forked-jkb49?file=/demo.tsx:782-988

Screen Shot 2021-03-10 at 2 00 39 PM

File: src/core/styles/common/constants/black.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export enum Black {
```
Comment (ovalenzuela19) [inline src/core/styles/common/constants/black.ts:1]: General question: Do we need these constants defined? Aren't these supposed to be defined in the `appTheme`?

Comment (tihuan) [reply src/core/styles/common/constants/black.ts:1]: Ahh that's right 😆 ! Totally forgot about that lol Thanks for catching this!

Yeah we will need to pass `theme` to `Emotion` and then use it for styling!

We can do it after Create-a-thon when we have time to review everything!

**Review by jshoe — APPROVED** [review]
Wow, nice!

**Pull request merged**